### PR TITLE
Bug Fix : OpenAPI file generated for the proto file is not valid

### LINKED ISSUE #1213: The openAPI generated from the proto file is not a valid openAPI
### Description:
In the gRPC scenario, the protobuf generates an OpenAPI mapping. This openAPI is not valid according to the OA specification. 

### Steps to reproduce:
generate a gRPC API.
validate the created OpenAPI mapping located inside <project>/target/gRPCSrc/OpenAPI s directory.

### Affected Product Version:
3.1.0

### Environment details (with versions):
- OS: MacOS Mojave
- Client:
- Env (Docker/K8s):

---

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Purpose
Fixed the proto to openAPI mapping such that the outcome is a valid openAPI.

- Added the missing responses object for the resource
- make sure that the resource starts with "/".

### Issues

Fixes #1213 

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

MacOS Mojave

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java [hunk 1]
```diff
@@ -104,7 +106,14 @@ void addOpenAPIPath(String path, String[] scopes, String throttlingTier) {
         if (openAPI.getPaths() == null) {
             openAPI.setPaths(new Paths());
         }
-        openAPI.getPaths().addPathItem(path, pathItem);
+        //as Responses object is mandatory
+        ApiResponse apiResponse = new ApiResponse();
+        apiResponse.setDescription("gRPC response");
```
Comment (Rajith90) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java:111]: Lets use constants

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java [hunk 2]
```diff
@@ -104,7 +106,14 @@ void addOpenAPIPath(String path, String[] scopes, String throttlingTier) {
...
+        ApiResponses apiResponses = new ApiResponses();
+        apiResponses.addApiResponse("200", apiResponse);
```
Comment (Rajith90) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java:113]: Lets use http constant status codes

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java [hunk 3]
```diff
@@ -104,7 +106,14 @@ void addOpenAPIPath(String path, String[] scopes, String throttlingTier) {
...
+        operation.setResponses(apiResponses);
+        //append forward slash to preserve openAPI syntax
+        openAPI.getPaths().addPathItem("/" + path, pathItem);
```
Comment (Rajith90) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java:116]: Lets use constants

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java (hunk 1 above)
Comment (VirajSalaka) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java:111]: This is a description. So do we still need to have a constant?

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/ProtoToOpenAPIConstants.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Rajith90) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/ProtoToOpenAPIConstants.java:2]: Wrong license year

Comment (VirajSalaka) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/ProtoToOpenAPIConstants.java]: updated via https://github.com/wso2/product-microgateway/pull/1214/commits/22f9e104d7bcd0e0652a2a82c725ff449820527f

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java (hunk 3 above)
Comment (VirajSalaka) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java]: updated via https://github.com/wso2/product-microgateway/pull/1214/commits/abc899d72f6c15dc0996fa057fe99cc26ffb10af

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java (hunk 2 above)
Comment (VirajSalaka) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java]: updated via https://github.com/wso2/product-microgateway/pull/1214/commits/abc899d72f6c15dc0996fa057fe99cc26ffb10af

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java (hunk 1 above)
Comment (VirajSalaka) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java]: updated via https://github.com/wso2/product-microgateway/pull/1214/commits/abc899d72f6c15dc0996fa057fe99cc26ffb10af

**Pull request merged**